client/components/ScopeDropdown/ScopeDropdown.tsx

```diff
@@ -63,10 +65,7 @@ const ScopeDropdown = (props: Props) => {
 	const { canManageCommunity, canManage } = scopeData.activePermissions;
 	const collectionSlug = locationData.params.collectionSlug || locationData.query.collectionSlug;
 	const pubSlug = locationData.params.pubSlug;
-	const nonActiveDashboardCollectionPub = getPrimaryOrFirstCollectionPub(
-		activePub,
-		communityData,
-	);
+	const nonActiveDashboardCollectionPub = getPrimaryOrFirstCollection(activePub, communityData);
```

Comment:
Given how much work this is doing, would you mind throwing it into a `useMemo()`?